### DISCUSSION TITLE
How to add nginx

### DISCUSSION THREAD

**Discussion by oie1** [leokhoa/laragon#410] [Q&A] [answered]
I'm not finding out how to add nginx or replace apache with it, how would I do it?

**Comment by oie1** [comment, marked as answer]
i found out, i have to add nginx folder to web server folder where apache is, im closing, i found out when i put mariadb folder where the mysql folder was


EDIT: i cant do it with apache to nginx, theres a seperate nginx folder but that dosent show up in laragon
edit wharver i did it in config

**Comment by rescenic** [comment]
Just make folder nginx in bin folder. And place nginx there.

**Comment by anchan42** [comment]
![laragon](https://github.com/leokhoa/laragon/assets/19514416/3c612786-ade1-4272-91db-726cda2b13c5)

Go to preference and tick the Nginx box.

**Comment by Qoraiche** [comment]
Thanks !
